Automattic/woocommerce-payments
# Handle the case when some attributes from WC's currency data is null.

Fixes #4001

#### Changes proposed in this Pull Request

Make sure we use empty string when some of the attributes from WC's currency data are null.

I put more detailed of the problem and my investigation in the issue.

Since this PR is fixing a problem that hasn't been released yet, I do not need to add a changelog (right?).

#### Testing instructions

0. Use WC 4.8.
1. Purchase a subscription product.
2. Open the payment detail page of that order.

With head branch, expect no error.
With base branch, expect an error.

Also, expect green E2E tests for WC 4.8.
With base branch, `WC - 4.8.0 | subscriptions - merchant` [failed](https://github.com/Automattic/woocommerce-payments/runs/5622680470?check_suite_focus=true).

-------------------


**Post merge**
